kytos/core/controller.py

```diff
@@ -157,9 +157,38 @@ def __init__(self, options=None, loop=None):
 
     def enable_logs(self):
         """Register kytos log and enable the logs."""
+        decorators = self.options.logger_decorators
+        decorators = [self._resolve(decorator) for decorator in decorators]
+        LogManager.decorate_logger_class(*decorators)
         LogManager.load_config_file(self.options.logging, self.options.debug)
         LogManager.enable_websocket(self.api_server.server)
         self.log = logging.getLogger(__name__)
+        self._reload_core()
+
+    @staticmethod
+    def _resolve(name):
+        """Resolve a dotted name to a global object."""
+        name = name.split('.')
+        used = name.pop(0)
+        found = __import__(used)
+        for name_part in name:
+            used = used + '.' + name_part
+            try:
+                found = getattr(found, name_part)
+            except AttributeError:
+                __import__(used)
+                found = getattr(found, name_part)
+        return found
+
+    @staticmethod
+    def _reload_core():
```

Comment:
@Ktmi, it's great how this staticmethod turned out, however its name has lost its original semantics since it's not reloading modules anymore. Could you update it's name to reflect more closely what it does? Something like `_patch_kytos_core_loggers` or something similar?

Reply:
Renamed the function to `_patch_core_loggers`.